examples/boids/src/main.rs

```diff
@@ -339,7 +339,8 @@ fn boids() {
         optional_features: wgpu::Features::default(),
         base_test_parameters: wgpu_test::TestParameters::default()
             .downlevel_flags(wgpu::DownlevelFlags::COMPUTE_SHADERS)
-            .limits(wgpu::Limits::downlevel_defaults()),
+            .limits(wgpu::Limits::downlevel_defaults())
```

Comment:
do we know why boids doesn't work for moltenvk on mac?

Reply:
There's a slew of validation errors related to early deletion of data, I believe it's related to https://github.com/gfx-rs/wgpu/issues/3160